gui/GameConqueror.py

```diff
@@ -905,7 +907,7 @@ def update_scan_result(self):
             # temporarily disable model for scanresult_liststore for the sake of performance
             self.scanresult_liststore.clear()
             for line in lines:
-                line = str(line)
+                line = str(line.decode("utf-8"))
```

Comment:
I agree with `.decode()` without "utf-8" but your other remark is definitely not the case. This is with Python 2 without the `str()`:
````
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/share/gameconqueror/GameConqueror.py", line 889, in scan_thread_func
    self.update_scan_result()
  File "/usr/share/gameconqueror/GameConqueror.py", line 912, in update_scan_result
    (a, o, rt, v, t) = list(map(str.strip, line.split(',')[:5]))
TypeError: descriptor 'strip' requires a 'str' object but received a 'unicode'
````